docs/_templates/include/schedule2017.rst

```diff
@@ -32,20 +32,10 @@
         <div class="col-xs-2">
           <p>{{ talk.Time }}</p>
         </div>
-        <div class="col-xs-10>
-          <p class=">
+        <div style="color: black;" class="col-xs-10">
+          <p>
 
-          {% if talk.Session in dont_link %}
```

Comment:
We need to build a proper way to handle this, likely pre-processing the JSON before checking it into the repo.